va_explorer/va_data_management/utils/coding.py

```diff
@@ -105,15 +106,31 @@ def run_coding_algorithms():
     # Load all verbal autopsies that don't have a cause coding
     # TODO: This should eventually check to see that there's a cause coding for
     # every supported algorithm
-    verbal_autopsies_without_causes = list(
-        VerbalAutopsy.objects.filter(causes__isnull=True)
-    )
 
     print(f"ALGORITHM SETTINGS: {ALGORITHM_SETTINGS}")
 
-    causes, issues = run_interva5(verbal_autopsies_without_causes)
+    causes_list = []
+    issues_list = []
+    verbal_autopsies_without_causes_list = []
 
-    return verbal_autopsies_without_causes, causes, issues
+    # Determine number of iterations necessary for large lists of verbal autopsies
+    count = VerbalAutopsy.objects.filter(causes__isnull=True).count()
+    batches = ceil(count / 5)
+    batch_size = 500
```

Comment:
```suggestion
    batch_size = 500
    batches = ceil(count / batch_size)
```

Reply:
This suggestion fixes a bug where `count / 5` leads to too many batches or runaway batch_start:batch_end in the loop.